app/models/permission.rb

```diff
@@ -1,12 +1,12 @@
 class Permission
   ADMIN = "CAN EDIT ABET"
-  ASSESSMENTS = "CAN EDIT ASSESSMENTS"
+  ASSIGNMENTS = "CAN EDIT ASSIGNMENTS"
```

Comment:
"CAN EDIT ASSESSMENTS" is how the permission is set in the external system we get permissions from. We may want to rename the constant but I think we need to keep the string value it's set to for integration purposes.